### PR TITLE
issue-1160: expand less compiler for core

### PR DESCRIPTION
**Pull request by oliverfoster**
https://github.com/adaptlearning/adapt_framework/issues/1160

* allow nested folders in preparation to include theme basic less

### CODE REVIEW

**Comment by tomgreenfield** [comment]
+1

File: grunt/config/less.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ module.exports = function (grunt, options) {
...
                 baseUrl: '<%= sourcedir %>',
                 mandatory: [
-                    '<%= sourcedir %>core/less/*.less'
+                    '<%= sourcedir %>core/less/**/*.less'
                 ],
                 src: [
-                    '<%= sourcedir %>menu/<%= menu %>/**/*.less',
                     '<%= sourcedir %>components/**/*.less',
                     '<%= sourcedir %>extensions/**/*.less',
+                    '<%= sourcedir %>menu/<%= menu %>/**/*.less',
```
Comment (tomgreenfield) [inline grunt/config/less.js]: My +1 still stands but we may want to discuss at some point whether extensions should appear _after_ the menu in the compile order.

**Comment by lc-thomasberger** [comment]
+1

File: grunt/config/less.js (hunk 1 above)
Comment (oliverfoster) [inline grunt/config/less.js]: yea, not sure if it's an issue or not, but we'll see i'm sure

**Comment by guywillis** [comment]
+1

**Pull request merged**